charts/mc-router/templates/service.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ include "mc-router.fullname" . }}
+  labels:
+    {{- include "mc-router.labels" . | nindent 4 }}
+spec:
+  type: {{ .Values.service.type }}
+  ports:
+    - port: {{ .Values.service.apiPort }}
+      {{- if list "LoadBalancer" "NodePort" | has .Values.service.type }}
+      nodePort: {{ .Values.service.apiPort }}
+      {{- end }}
+      targetPort: api
+      protocol: TCP
+      name: api
```

Comment:
It would be good to allow for API port to be optionally exposed. In fact, the API might be need to be declared in a separate service object since they may want to use `type: ClusterIP` for the API to keep it internal, but `type: NodePort` for the main routing port to expose externally.

Reply:
Done!